main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <ctype.h>
+#include "quadratic_equation.h"
+
+int main() {
+    double first_par = 0;
+    double second_par = 0;
+    double third_par = 0;
+
+    printf("Enter the first second and third parameters of the quadratic equation separated by a space:\n");
+
+    int result_scanf = 0;

```

Comment:
Not really informative variable name. What is result_scanf? The variable holds the number of correctly entered parameters - the name is supposed to reflect this.